index.html

```diff
@@ -36,6 +36,131 @@ <h1 class="heading-1">
+            <img src="img/Rectangle22.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button" >
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span> 
+        </button>
+        </div>
+        <div class="background-color">
+            <img src="img/Rectangle23.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button">
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span> 
+        </button>
+        </div>
+        <div class="background-color">
+            <img src="img/Rectangle24.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button">
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span> 
+        </button>
+        </div>
+        <div class="background-color">
+            <img src="img/Rectangle25.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button">
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span> 
```

Comment:
- [ ] The button (e.g., "See This project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), kindly make see this project as a link or button with a hover color.